db_util.sql

```diff
@@ -0,0 +1,57 @@
+-- db_util.sql
+/*
+psql -d postgres -f ./db_util.sql ====> creates db and tables
+psql -d pc ==========> opens up db in terminal
+psql -f [file.sql] -U ubuntu -d [dbname]
+*/
+
+DROP DATABASE IF EXISTS pc;
+CREATE DATABASE pc;
+
+-- single line comment
+/* multi line comment */
+
+\c pc;
+
+CREATE TABLE users (
+    ID SERIAL PRIMARY KEY,
+    username VARCHAR (100),
+    password VARCHAR (100)
```

Comment:
a "role" attribute would be really useful.

Reply:
will do